inventory/node.toml

```diff
@@ -6909,6 +6944,13 @@ arch = "linux-x64"
 url = "https://s3.amazonaws.com/heroku-nodebin/node/staging/linux-x64/node-v12.20.0-linux-x64.tar.gz"
 etag = "3383e1624171a83a04740f24831c5c92-3"
 
+[[releases]]
+version = "12.20.1"
+channel = "staging"
+arch = "linux-x64"
+url = "https://s3.amazonaws.com/heroku-nodebin/node/staging/linux-x64/node-v12.20.1-linux-x64.tar.gz"
+etag = "51c80c3820855d8e77f37c9a67b9d89b-3"
+
```

Comment:
These additions to the inventory too - it looks like something happened with the git history that needs to be fixed, or it could just need a rebase.